course_info/tests/jasmine/controllers/SearchControllerSpec.js

```diff
@@ -90,4 +90,30 @@ describe('Unit testing SearchController', function() {
     describe('$scope.searchCourseInstances()', function() {
 
     });
+
+    describe('getCourseDescription()', function() {
+        it('should display the course title if title is present', function () {
+            var course = { title: 'ABC'};
+            result = scope.getCourseDescription(course);
+            expect(result).toEqual('ABC');
+        });
+
+        it('should dipslay the course short title if title is not present and short title is', function() {
+            var course = { title: '', short_title: 'short'};
```

Comment:
Should probably also check that if both title and short_title are present, title is preferred.